recipes/sos-rq/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sos-rq" %}
+{% set version = "0.9.16.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "8c8a1093a0fe74b835c8d8d731426ed38268e7ffd59b2bdae2f3ad10f98324f3" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: TRUE # [not py36]
+  script: python -m pip install --no-deps --ignore-installed . --verbose 
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - rq
```

Comment:
Can `rq` be removed as host dependecy?

Reply:
`host` dependency? This looks like a run dependency.